tqp/flask.py

```diff
@@ -43,3 +46,23 @@ def handle_message(self, msg, payload):
         with self.app.app_context():
             _get_tqp_context()[CTX_PAYLOAD_KEY] = payload
             super().handle_message(msg, payload)
+
+    def set_log_formatter(self, get_message_id: None):
+        class PollerFormatter(logging.Formatter):
```

Comment:
Is it really necessary to close over instantiating this class? Can `get_message_id` be passed in as a ctor arg?